waspc/src/Wasp/Generator/DbGenerator.hs

```diff
@@ -63,11 +63,11 @@ genPrismaSchema ::
   Generator FileDraft
 genPrismaSchema spec = do
   (datasourceProvider :: String) <- case dbSystem of
-    AS.Db.PostgreSQL -> return "postgresql"
```

Comment:
Again using it for Prisma, ok.

Btw note that if you are to define somewhere in our codebase which string does Prisma use as identifier for postgre, you don't have to do it in AppSpec part of our codebase. It has nothign to do with Wasp or AppSpec, it is Prisma-specific thing. I would instead expect to see such definition somewhere in the part of our codebase where we deine Prisma specific stuff (PSL).